src/main/resources/db/migration/R__functions.sql

```diff
@@ -200,29 +196,23 @@ ORDER BY url;
 
 -- Currently existing log of object changes
 CREATE OR REPLACE VIEW current_log AS
-SELECT
-    ol.id,
-    ol.version_id,
-    operation,
-    CASE operation
-        WHEN 'DEL' THEN old_o.url
-        WHEN 'INS' THEN new_o.url
-        WHEN 'UPD' THEN new_o.url
-    END AS url,
-    CASE operation
-        WHEN 'DEL' THEN old_o.hash
-        WHEN 'INS' THEN NULL
-        WHEN 'UPD' THEN old_o.hash
-    END AS old_hash,
-    CASE operation
-        WHEN 'DEL' THEN NULL
-        WHEN 'INS' THEN new_o.content
-        WHEN 'UPD' THEN new_o.content
-    END AS content
-FROM object_log ol
-LEFT JOIN objects old_o ON old_o.id = ol.old_object_id
-LEFT JOIN objects new_o ON new_o.id = ol.new_object_id
-ORDER BY id ASC;
+    WITH combined_operations AS (
```

Comment:
This one is 4 times slower on the current prepdev database, from about 300 to about 1200ms. If we call `select * from current_log` in a loop like for delta at the start of the application, it's going to get significantly slower. If we don't, it's probably not an issue. Or it may happen that `changes_exist` is slower, for example.

Reply:
I haven't considered the startup loop. Maybe it is faster to solve multiple updates in Scala code and keep the query simple.

Reply:
Moving the combining of `object_log` entries to Scala is quite hard since the `changes_exist` must also do this, and this one is invoked from `freeze_version`. So if we want to move this to Scala we'd also need to (partially) move `freeze_version`.

Reply:
Fixed the performance by modifying `read_delta` not to join this view to the `versions` table. This dropped the query time from ~700ms to ~10ms again (on my laptop).